.github/release-drafter-config.yml

```
@@ -18,6 +18,10 @@ categories:
   - title: 'Breaking Changes'
     labels:
       - 'breakingchange'
+
+  - title: '🧪 Experimental Features'
+    labels:
+      - 'experimental'
```

There is no empty line in this block.
```suggestion
  - title: '🧪 Experimental Features'
    labels:
      - 'experimental'
```